documentation/decisions/adr/000N-video-conferencing.md

```diff
@@ -0,0 +1,166 @@
+- [Microsoft Teams](https://www.microsoft.com/en-us/microsoft-teams/group-chat-software) - *decision* because of  
+- [Google Meet](https://meet.google.com/) - *decision* because of  
+- [Jitsu](https://meet.jit.si/) - *decision* because of  
+
+## Decision Outcome <!-- REQUIRED -->
+
+TBD 
+
+### Positive Consequences <!-- OPTIONAL -->
+
+- **Improved Communication**: Video conferencing enables real-tme face-to-face communication, fostering better understanding, transparency, and collaboration among team members, stakeholders, and the general public.
+- **Enhanced Collaboration**: With screen sharing and chat features, participants can work together on documents, brainstorm ideas, and address issues efficiently, enhancing overall teamwork.
+- **Extended Reach**: The tool allows you to connect with participants from different locations, making it easier to work with remote teams, partners, and the public.
+- **Flexibility and Convenience**: Participants can join meetings from their preferred location, whether it's their office, home, or while on the go, promoting flexibility and convenience.
+- **Accessibility**: Video conferencing tools often offer closed captioning and accessibility features, making meetings more inclusive for individuals with impairments.
+
+### Negative Consequences <!-- OPTIONAL -->
+
+- **Technical Challenges**: Technical issues, such as poor internet connectivity, audio/video glitches, and compatibility problems, can disrupt meetings and lead to frustration.
+- **Security Concerns**: Video conferencing tools may pose security risks, like unauthorized access, data breaches, or potential privacy violations if not properly configured and managed.
+- **Fatigue and Burnout**: Frequent video meetings can lead to "Zoom fatigue" or virtual meeting burnout, affecting participant engagement and focus.
+- **Distractions and Multitasking**: Participants may be prone to distractions or multitasking during virtual meetings, reducing attention and active engagement.
+- **Learning Curve**: New users may find it challenging to navigate and fully utilize the features of the video conference tool, leading to a learning curve for some team members.
+- **Bandwidth Consumption**: Video conferencing consumes significant internet bandwidth, which may impact other online activities if the network is not robust enough. This could also be limiting for individuals with low or slow broadband connection.
+
+### Back-up Options
+
+
+
+## Comparison Matrix
+
+- ✅ Feature available, meets requirement
+- ❌ Feature not available, does not meet requirement
+- 🔄 Partial feature, limited feature availability, feature in progress or undergoing improvements
+- 1-3 Strength level
+- ❓Unknown
+
+| Factor                      | Zoom | Microsoft Teams | Google Meet | Jitsu |
+| --------------------------- | :--------: | :--------: | :---------: | :-----: | 
+| Public Access                   |     ✅      |   ✅    |      ✅      |    ✅    |  
```

Comment:
Nit: depending on the IDE/text editor you use, there might be a plugin available that automatically formats these tables to have the same column width in the Markdown text itself.